src/OrchardCore/OrchardCore.Data.YesSql.Abstractions/DbConnectionValidatorResult.cs

```diff
@@ -1,22 +1,21 @@
 namespace OrchardCore.Data;
 
+/// <summary>
+/// 'Unknown' indicates that the connection string status is unknown or was not validated.
+/// 'NoProvider' indicated that the database provider is missing.
+/// 'DocumentTableNotFound' indicates that the connection string was valid, while the Document table does not exists.
+/// 'DocumentFound' indicates that the connection string was valid, while the Document table exists.
+/// 'ShellDescriptorDocumentFound' indicates that the connection string was valid and the document table exists with at least one shell-descriptor document.
+/// 'InvalidConnection' unable to open a connection to the given connection string.
+/// 'UnsupportedProvider' indicates invalid or unsupported database provider.
+/// </summary>
```

Comment:
Sorry I meant to replace the simple comments with a `<summary>` tag + a single line of comment, but still one per value below, then here also at the top a global `<summary>` tag + a single line of comment.

So that you have the intellisense for both the enum type and their individual values.